app/Http/Resources/CategoryCollection.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
Added CategoryCollections to help with the endpoint JSON formatting. If using the CategoryResource direct I find it breaks some of the default laravel pagination helpers.